worldengine/model/world.py

```diff
@@ -852,7 +851,67 @@ def is_chaparral(self, pos):
     #
 
     def n_actual_plates(self):
-        return self.layers['plates'].data.max() + 1
+        return self.plates.data.max() + 1
+
+    #
+    # Accessors
+    #
+
+    @property
+    def elevation(self):
+        return self.layers['elevation']
```

Comment:
Yes, it is possible to have different behavior looking at the type of the value and yes, we could pass a tutple to the property setter but I am very convinced this would cause confusion. I would strongly prefer using explicit setters as we have now. Anyway setters are used very rarely, while property getters are used much more frequently, so they are much more important